data.sql

```diff
@@ -9,3 +9,13 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animalsSET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
+INSERT INTO visits (animal_name, vet_id, visit_date) VALUES('Agumon', 1, '2020-05-24'), ('Agumon', 3, '2020-07-22'), ('Gabumon', 4, '2021-02-02'), ('Pikachu', 2, '2020-01-05'), ('Pikachu', 2, '2020-03-08'), ('Pikachu', 2, '2020-05-14'), ('Devimon', 3, '2021-05-04'), ('Charmander', 4, '2021-02-24'), ('Plantmon', 2, '2019-12-21'), ('Plantmon', 1, '2020-08-10'), ('Plantmon', 2, '2021-04-07'), ('Squirtle', 3, '2019-09-29'),
+('Angemon', 4, '2020-10-03'), ('Angemon', 4, '2020-11-04'), ('Boarmon', 2, '2019-01-24'), ('Boarmon', 2, '2019-05-15'), ('Boarmon', 2, '2020-02-27'), ('Boarmon', 2, '2020-08-03'), ('Blossom', 3, '2020-05-24'), ('Blossom', 1, '2021-01-11');
+INSERT INTO specializations (vet_id, species) VALUES (1, 'Pokemon'), (3, 'Digimon'), (3, 'Pokemon'), (4, 'Digimon');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23'), ('Maisy Smith', 26, '2019-01-17'), ('Stephanie Mendez', 64, '1981-05-04'), ('Jack Harkness', 38, '2008-06-08');
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
```

Comment:
> Find a way to decrease the execution time of the first query. Look for hints in the previous lessons.

- [ ] Looking at the requirement, you are meant to find a way to reduce the execution time. but in your screenshots, the execution time for the after improvements, are higher than the time before it. I'll suggest you actually work with your time on the improvement, You can consider adding `indexes` to the id as it will improve the performance and in turn, reduce execution time. :+1: :+1: 

## Your feedback to the previous reviewer is valid, but now there's a task that doesn't require it to be done alone on your PC, your partners can work on the improvement and when they are done, you can pull the changes, and verify it on your PC since it's your database that has the data. :+1: